recipes/BulkEval.py

```
@@ -379,41 +524,141 @@ def submit(
     response: BulkEvalPage.ResponseModel,
 ) -> list[Future[TaskResult]]:
     futs = []
+    for ep in request.eval_prompts or []:
+        validate_eval_prompt(ep)
+    try:
+        model_spec = AIModelSpec.objects.get(name=request.selected_model)
+    except AIModelSpec.DoesNotExist:
+        raise UserError(f"Model {request.selected_model!r} not found.")
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)